pybamm/install_odes.py

```diff
@@ -5,46 +5,46 @@
 import sys
 import logging
 import subprocess
+from importlib import import_module
 
 from pybamm.util import root_dir
 
-try:
-    # wget module is required to download SUNDIALS or SuiteSparse.
-    import wget
+if sys.platform == "win32":
+    raise Exception("pybamm_install_odes is not supported on Windows.")
 
-    NO_WGET = False
-except ModuleNotFoundError:
-    NO_WGET = True
+SUNDIALS_VERSION = "6.5.0"
 
+def install_required_module(module):
+    try:
+        import_module(module)
+    except ModuleNotFoundError:
+        print(f"{module} module not found. Installing {module}...")
+        subprocess.run(["pip", "install", module], check=True)
```

Comment:
I know I requested adding this earlier, but I think modifying someone's virtual environment in such an impromptu way seems weird. I suggest we should let go of this function and throw an error on `wget` and `CMake` not being installed instead so that users can do that on their own (but add to the documentation that the script requires them). I am working on a branch locally where `urllib` from the standard library will be used to download SUNDIALS (we can let go of `wget` then). Is it fine if I push the same changes to your branch after the work on your end is complete?

P.S. The command would have failed anyway if someone would have not had `pip` but `pip3` as their shell-completion or if they used `python3 -m pip` instead of `pip`.

Reply:
> Is it fine if I push the same changes to your branch after the work on your end is complete?

Yeah, no worries I'd let you know once i finish up with my changes